### ISSUE TITLE
npm v16: ERR_NO_CRYPTO

### ISSUE THREAD

**Issue by olof-nord** [carlosedp/riscv-bringup#36]
In the prebuilt [node-v16.0.0-linux-riscv64.tar.gz](https://github.com/carlosedp/riscv-bringup/releases/download/v1.0/node-v16.0.0-linux-riscv64.tar.gz) release asset, the npm build shows the below error message:

```shell
➜  ~ npm --version
Error [ERR_NO_CRYPTO]: Node.js is not compiled with OpenSSL crypto support
    at new NodeError (node:internal/errors:329:5)
    at assertCrypto (node:internal/util:128:11)
    at node:crypto:36:1
    at NativeModule.compileForInternalLoader (node:internal/bootstrap/loaders:297:7)
    at NativeModule.compileForPublicLoader (node:internal/bootstrap/loaders:237:10)
    at loadNativeModule (node:internal/modules/cjs/helpers:39:9)
    at Function.Module._load (node:internal/modules/cjs/loader:787:15)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/home/ubuntu/programs/node-v16.0.0-linux-riscv64/lib/node_modules/npm/node_modules/ssri/index.js:3:16)
/home/ubuntu/programs/node-v16.0.0-linux-riscv64/lib/node_modules/npm/node_modules/@npmcli/config/lib/index.js:163
      throw new Error('call config.load() before reading values')
      ^

Error: call config.load() before reading values
    at Config.get (/home/ubuntu/programs/node-v16.0.0-linux-riscv64/lib/node_modules/npm/node_modules/@npmcli/config/lib/index.js:163:13)
    at process.errorHandler (/home/ubuntu/programs/node-v16.0.0-linux-riscv64/lib/node_modules/npm/lib/utils/error-handler.js:179:32)
    at process.emit (node:events:369:20)
    at process._fatalException (node:internal/process/execution:163:25)
➜  ~
```

Running on Ubuntu Server 5.13.0-1017-generic HiFive Unmatched

**Comment by carlosedp** [comment]
Ah yes, it was a gap in v16 when I built it. Gonna try to rebuild it soon.

**Comment by carlosedp** [comment]
Fixed by #37
